### PR TITLE
clean: Improve meta descriptions of top searched pages DOCS-227

### PR DESCRIPTION
**Pull request by prcr**
Review and curate the meta descriptions of the [top landing pages for Google searches](https://docs.google.com/spreadsheets/d/191E8aGjiJwpSp8sroPZQUeZMmSiJ7t2HHLFGnJvkhEE/edit#gid=566714567).

If possible also include the changes to the submodules in this pull request:

- https://github.com/codacy/chart/pull/681
- https://github.com/codacy/codacy-coverage-reporter/pull/314

### CODE REVIEW

File: docs/faq/repositories/how-do-i-reanalyze-my-repository.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # How do I reanalyze my repository?
 
-Click on the latest commit from your list in the **Commit** view and click to reanalyze as the latest commit includes all the code of all the PRs.
+Open the latest commit on the [Commits page](../../repositories/commits.md) and click the button next to **Analyzed**, as the latest commit includes all the code from all the pull requests.
```
Comment (mrfyda) [inline docs/faq/repositories/how-do-i-reanalyze-my-repository.md:3]: a comment not related to this change, but I don't think this sentence is accurate:

> the latest commit includes all the code from all the pull requests

it depends on the target branch of the pull request, and reanalyzing the latest commit does not automatically reanalyze the pull requests results.

File: docs/repositories-configure/codacy-configuration-file.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-description: The Codacy configuration file supports configuring certain advanced features on Codacy.
+description: Use the Codacy configuration file to configure advanced features on Codacy with more control such as ignoring files for duplication or a specific tool, configuring the root directory to start the analysis, and adding custom extensions to languages.
```
Comment (mrfyda) [inline docs/repositories-configure/codacy-configuration-file.md:2]:
```suggestion
description: Use the Codacy configuration file to configure advanced features on Codacy with more control such as ignoring files for duplication or a specific tool, configuring the root directory to start the analysis, and adding custom file extensions to languages.
```

**Review by mrfyda — APPROVED** [review]

File: docs/repositories-configure/codacy-configuration-file.md (hunk 2 above)
Comment (prcr) [reply docs/repositories-configure/codacy-configuration-file.md]: Nice clarification! I applied the same change to the introduction.

File: docs/faq/repositories/how-do-i-reanalyze-my-repository.md (hunk 1 above)
Comment (prcr) [reply docs/faq/repositories/how-do-i-reanalyze-my-repository.md:3]: I'm going to open a separate pull request to fix this. :+1:

**Pull request merged**